### ISSUE TITLE
Tickets not received into FreshDesk + No GCcollab emails received --- many domains

### ISSUE THREAD

**Issue by Menrika** [gctools-outilsgc/gcconnex#1975]
## HelpDesk has been unable to receive Freshdesk tickets from some users of over multiple organizations even when it had been submitted. Users are also not receiving emails from **GCcollab.** 

It is suspected that their department network is blocking the emails.

Issue raised with : **21** users in which 2 were not associated with accounts 

Excel sheet: https://gccollab.gctools-outilsgc.ca/helpdesk/attachments/2107975546

Emails had to be manually validated by Helpdesk

## Ticket : https://gccollab.gctools-outilsgc.ca/a/tickets/7099

Refer to CRA and TSBC issue with email validations : https://github.com/gctools-outilsgc/gcconnex/issues/1954

## For the development team
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed

**Comment by martydonofrio** [comment]
Here are the orgs that reported email issues so far:

University of   Ottawa - university - student
Canada Revenue Agency
Fisheries and Oceans Canada
Immigration, Refugees and Citizenship   Canada
Delmar International Inc. - business
National Defence
University of Waterloo - university -   student
Canada Border Services Agency
Auditor General of Canada (Office of the)
MacEwan University - university - student
Employment and Social Development Canada
Global Affairs Canada
National Defence / Defense nationale
Canadian Institutes of Health Research
Canada Post
Immigration and Refugee Board of Canada
Public Safety Canada

**Comment by bryan-robitaille** [comment]
I've made some modifications on SendGrid to rewrite all tracking links to remove any mention of sendgrid.net

**Comment by ethanWallace** [comment]
Given the original information provided about the issue. If a user received a success message while submitting a ticket, the API functioned correctly and anything not working after that is out of our control. Even though it won't solve this issue directly, a pull request has been made to improve the system feedback when submitting a ticket on all platforms.

@martydonofrio Have you contacted Freshdesk about this issue and are users still not recieving emails after the change Brian made?

**Comment by nschonni** [comment]
At IRCC, I have received GCMessage emails for private message notifications, but still don't receive other GCCollab messages

**Comment by martydonofrio** [comment]
@ethanWallace We are still getting reports of users not getting confirmation emails. I sent a message to freshdesk asking for help and will let you know what happens.

**Comment by martydonofrio** [comment]
We are now getting reports of some groups not sending out their notifications at all.

**Comment by martydonofrio** [comment]
Here is an example of a group that is not getting notifications through email:
https://gccollab.ca/groups/profile/595477/encanadian-cubesat-projectfr

**Comment by martydonofrio** [comment]
I tested receiving tickets with 5 people at IRCC through the freshdesk site (https://gccollab.gctools-outilsgc.ca/en/support/tickets/new) instead of the help page and all tickets went through. All my responses were received too.

**Comment by LemieuxGen** [comment]
Blocked: Currently beyond our control. Management needs to decide how we want to proceed with this issue. Options could be: 
1- SMTP server in the cloud
2- comms to CIOS
3- private vendors directly to unblock list SendGrid. 

@ethanWallace to divide card in separate cards that could potentially be addressed separately.

**Comment by martydonofrio** [comment]
2 more groups reported not receiving any notifications:

https://gccollab.ca/groups/profile/1503270/ensciencefridayfr
https://gccollab.ca/groups/profile/1597275/enfederal-water-networkfr

**Comment by Menrika** [comment]
Another Ticket to check, in which their security group responded : https://gccollab.gctools-outilsgc.ca/a/tickets/7681

**Comment by Menrika** [comment]
It appears that a lot of employees (if not most) from Financial Consumer Agency of Canada are also dealing with this issue. 
https://gccollab.gctools-outilsgc.ca/a/tickets/7880

**Comment by Menrika** [comment]
Tickets not received into Freshdesk : jean.richer@cic.gc.ca (IRCC)

- also getting reports that password reset emails are not being received.

https://gccollab.gctools-outilsgc.ca/a/tickets/8411

**Comment by Menrika** [comment]
https://gccollab.gctools-outilsgc.ca/a/tickets/8464 ( ESDC )

**Comment by marc-donofrio** [comment]
https://gccollab.gctools-outilsgc.ca/a/tickets/9009

**Comment by nschonni** [comment]
Just received a few digest emails at IRCC this morning (9:06, 9:07, and 9:11), so maybe part of the IRCC is resolved
